cli_tools/common/disk/inspect.go

```diff
@@ -60,14 +60,14 @@ func NewInspector(env daisycommon.EnvironmentSettings, logger logging.Logger) (I
 		logPrefix += "-"
 	}
 	wf.Name = logPrefix + "inspect"
-
-	return &bootInspector{daisycommon.NewDaisyWorker(wf, env, logger), logger}, nil
+	rl := daisyutils.NewResourceLabeler(wf.Name, env.ExecutionID, env.Labels, env.StorageLocation)
+	return &bootInspector{daisyutils.NewDaisyWorker(wf, env, logger, rl), logger}, nil
```

Comment:
@dntczdx 
> It looks like "resource labeling" is added in some places - not simply refactoring but added new actions. Is that the case? Is that by design?

Thanks, that's a great call-out: I removed that here since you're right I didn't intend to add new features.